internal/api/helper.go

```diff
@@ -0,0 +1,22 @@
+package api
```

Comment:
we can and should move this in 
```
api/numaresourcesoperator/v1/helper/
```
we can call it `namespacedname` (`package namespacedname`)
packages in `.../helper` build on top of the main api package, so we can add deps (in a reasonnable fashion) here.

Reply:
please add the copyright boilerplate (`hack/boilerplate.go.txt`)